src/api/users/users.get.test.ts

```diff
@@ -0,0 +1,80 @@
+import { after, before, describe, it } from 'node:test';
+import { prisma } from '@core/db/db.js';
+import axios from 'axios';
+import { expect } from 'chai';
+import jwt from 'jsonwebtoken';
+
+const USER_TO_CREATE = {
+  name: 'John Doe',
+  email: 'john.doe@example.com',
+  password: 'password123',
+  birthDate: '1990-01-01T00:00:00.000Z',
+};
+
+describe('Get User', () => {
+  before(async () => {
+    await prisma.user.create({
+      data: USER_TO_CREATE,
+    });
+  });
+
+  it('should return user', async () => {
+    const validToken = `Bearer ${jwt.sign({ userId: 'test-user-id' }, process.env.JWT_SECRET, { expiresIn: '1h' })}`;
+
+    const userId = (await prisma.user.findFirst({
+      where: {
+        email: USER_TO_CREATE.email,
+      },
+    }))!;
+
+    const response = await axios.get(`http://localhost:8080/users/${userId.id}`, {
+      headers: {
+        Authorization: validToken,
+      },
+    });
+
+    expect(response.status).to.equal(200);
+    expect(response.data.id).to.equal(userId.id);
+    expect(response.data.name).to.equal(USER_TO_CREATE.name);
+    expect(response.data.email).to.equal(USER_TO_CREATE.email);
+    expect(response.data.birthDate).to.equal(USER_TO_CREATE.birthDate);
```

Comment:
Importante usar `to.be.deep.eq` pra conferir body de resposta, ainda mais no REST que não tem uma filtragem de quais campos ele vai retornar. Assim qualquer campo e menos ou a mais que retorne, o teste falha, como deveria.